.github/workflows/integrations.yml

```diff
@@ -382,3 +382,49 @@ jobs:
         with:
           slack-webhook: ${{ secrets.E2E_INTEGRATIONS_SLACK_WEBHOOK }}
           job-status: ${{ needs.detectron2.result }}
+
+  rlang:
+    env:
+      NEPTUNE_PROJECT: e2e-tests/integrations
+    timeout-minutes: 75
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: macos-latest,   r: 'release'}
```

Comment:
Maybe Release + Some older version of R?

Reply:
Not sure if it is worth it. We have just a few users of this integration.

Reply:
Ok